kratos/tests/cpp_tests/processes/test_find_nodal_neighbours_process.cpp

```diff
@@ -0,0 +1,122 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:   BSD License
+//      Kratos default license: kratos/license.txt
```

Comment:
Format.

It also would be nice to take the chance to do an MPI test of this (I think @RiccardoRossi specifically changed the implementation because of this).